libvuln/migrations/08-updater-status.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS updater_status (
```

Comment:
nit: could you just format this sql :D

Reply:
This is the tool I use for this: https://github.com/darold/pgFormatter

Reply:
Thanks both, I ran the tool which fixed the inconsistent indents I missed, but it didn't prettify the formatting like some of the other migration files (in fact it would undo any manual prettifying). Let me know if you want me to put that in or if I could do something differently